code/modules/antagonists/traitor/objectives/demoralise_crew.dm

```diff
@@ -1,53 +1,7 @@
-#define MAX_CREW_RATIO 0.33
-#define MIN_CREW_DEMORALISED 8
-#define MAX_CREW_DEMORALISED 16
-
 /datum/traitor_objective_category/demoralise
 	name = "Demoralise Crew"
 	objectives = list(
-		/datum/traitor_objective/demoralise/poster = 2,
-		/datum/traitor_objective/demoralise/graffiti = 1,
+		/datum/traitor_objective/target_player/assault = 1,
+		/datum/traitor_objective/destroy_item/demoralise = 1,
 	)
 	weight = OBJECTIVE_WEIGHT_UNLIKELY
```

Comment:
This doesn't really need to be in its own file, we don't have a clear separation of objectives and categories so this is a bit complicated. I'll probably clean this up later

Reply:
Moved into one of the other files.